packages/manager/modules/nutanix/src/dashboard/general-info/redeploy/template.html

```diff
@@ -45,12 +45,12 @@ <h2 data-translate="nutanix_redeploy_title"></h2>
             <ul class="list-unstyled">
                 <li>
                     <span
-                        data-ng-bind=":: $ctrl.CLUSTER_CONFIG_TERMS.PRISM_CENTRAL + ' :'"
+                        data-translate="{{('nutanix_redeploy_prism_central' | translate) + ' :'}}"
                     ></span>
                     <ul>
                         <li class="align-bottom">
                             <span
-                                data-ng-bind=":: $ctrl.CLUSTER_CONFIG_TERMS.PRISM_CENTRAL_TYPE + ' :'"
+                                data-translate="{{('nutanix_redeploy_prism_central_type' | translate) + ' :'}}"
```

Comment:
Same here and others:
https://github.com/ovh/manager/pull/7478/files#r912957463

Reply:
Here I cannot move ':' from html to translation because I need nutanix_redeploy_prism_central_type without ':' in few more places in html.